solana/gatekeeper-lib/src/commands/add-gatekeeper.ts

```diff
@@ -20,8 +20,8 @@ export default class AddGatekeeper extends Command {
 
   static flags = {
     help: Flags.help({ char: "h" }),
-    gatekeeperKey: gatekeeperKeyFlag(),
-    gatekeeperNetworkKey: gatekeeperNetworkKeyFlag(),
+    authorityKeypair: authorityKeypairFlag(),
+    gatekeeperNetworkPublicKey: gatekeeperNetworkPublicKeyFlag(),
```

Comment:
Is that even used? It seems like the GKN public key comes from the authorityKeyPair. I know it's inconsistent right now but I would JUST call it `gatekeeperKeyPair` until we introduce the authority concept. Please add a comment here indicating that `eth` implements an authority abstraction